pandas/core/indexing.py

```diff
@@ -1,10 +1,16 @@
 # pylint: disable=W0223
```

Comment:
can you remove this warning? The general problem with these warnings is that they are added, but never removed.

Reply:
that's fine too